src/template/dto/template-content.dto.ts

```diff
@@ -0,0 +1,66 @@
+import { Exclude, Expose } from "class-transformer";
+import { ApiProperty, ApiPropertyOptional } from "@nestjs/swagger";
+
+export class TemplateContentDto {
```

Comment:
Please add a few words of description to each of the DTO attributes
cc @snehal0904

Reply:
@coolbung Done sir